release_docs/CHANGELOG.md

```diff
@@ -529,6 +562,17 @@ Added Fortran wrapper h5fdsubfiling_get_file_mapping_f() for the subfiling file
 
 ## Java Library
 
+### Added compound datatype support for Java FFM bindings
```

Comment:
This entire entry can probably just be removed, as FFM is entirely new anyway

Reply:
Done